openlibrary/templates/lists/widget.html

```diff
@@ -76,24 +76,25 @@ <h3 class="header">
 $if include_widget:
   $if render_once('lists/widget.i18n-strings'):
     $:i18n_input()
-  $:macros.ReadingLogDropper(user_lists, work=work, edition_key=(edition and edition.key), key=default_seed_key, users_work_read_status=users_work_read_status, page_url=page_url, async_load=async_load)
-  <ul id="already-lists" class="listLists already-lists">
-    $if async_load:
-        <div class="list-overview-loading-indicator">Loading<span class="loading-ellipsis">...</span></div>
-    $else:
-        $:render_already_lists(user_lists, user_key)
-  </ul>
+
+  $if 'my_books_dropper' in ctx.features:
+    $:render_template('my_books/dropper', page, async_load=async_load)
+  $else:
+    $:macros.ReadingLogDropper(user_lists, work=work, edition_key=(edition and edition.key), key=default_seed_key, users_work_read_status=users_work_read_status, page_url=page_url, async_load=async_load)
+
+  $if username and show_already_lists:
```

Comment:
BLOCKER:
Currently on search page we're not loading this which maybe we want to (we need some way to show that a book/work has been added to one of my lists).

Maybe bring to design call? Do we make this a `details` dropdown with fixed space?

Do we wait and use the checkbox approach? 

It _would_ be nice to have some indication that this book is on list(s) -- e.g. "On `Interesting` and 2 others"? 🤷

Reply:
For now, when we add a book to a list, can we just keep the list on the list-dropper with a ✔️ on it? If we click the item again, it will simply remove the ✔️

Reply:
Updated.  Lists can now be managed entirely within the dropper, and the active lists showcase has been removed.

![image](https://github.com/internetarchive/openlibrary/assets/28732543/e4ebfec2-bf40-4885-ba23-39c88803158c)